CRM/Upgrade/Incremental/php/FiveThirty.php

```diff
@@ -73,6 +73,8 @@ public function setPostUpgradeMessage(&$postUpgradeMessage, $rev) {
   public function upgrade_5_30_alpha1($rev) {
     $this->addTask(ts('Upgrade DB to %1: SQL', [1 => $rev]), 'runSql', $rev);
     $this->addTask('Add core (required) extension Financial ACLs', 'installFinancialAcls');
+    $this->addTask('core-issue#365 - Add created_date to civicrm_action_schedule', 'addColumn',
+      'civicrm_action_schedule', 'created_date', "timestamp NULL  DEFAULT CURRENT_TIMESTAMP COMMENT 'When was the schedule reminder created.'");
```

Comment:
Will the alter table statement that this creates not result in filling the value of each row with current time when the upgrade is run? That is what we want.

Reply:
Yes, I have rechecked on my local, after running the upgrade script and it fills the created_date with current time for existing reminders, which is correct.